scrapy_zyte_api/handler.py

```diff
@@ -86,6 +86,40 @@ def _prepare_api_params(self, request: Request) -> Optional[dict]:
             raise IgnoreRequest()
         return api_params
 
+    def _update_stats(self):
+        prefix = "scrapy-zyte-api"
+        for stat in (
+            '429',
+            'attempts',
+            'errors',
+            'extracted_queries',
+            'fatal_errors',
+            'input_queries',
+            'results',
+        ):
+            self._stats.set_value(
+                f"{prefix}/{stat}",
+                getattr(self._client.agg_stats, f"n_{stat}"),
+            )
+        for source, target in (
+            ('connect', 'connection'),
+            ('total', 'response'),
+        ):
+            self._stats.set_value(
+                f"{prefix}/mean_{target}_seconds",
+                getattr(self._client.agg_stats, f"time_{source}_stats").mean(),
+            )
+        for counter in (
+            'api_error_types',
+            'exception_types',
+            'status_codes',
+        ):
+            for key, value in getattr(self._client.agg_stats, counter).items():
+                self._stats.set_value(
+                    f"{prefix}/{counter}/{key}",
+                    value,
+                )
```

Comment:
What do you think about exposing success_ratio, throttle_ration, and maybe error_ratio as well? 

For error_ratio my concern is that the name might suggest it's just 1-success_ratio, but that's a very different statistic.

Reply:
Sounds good. I would probably fall on that trap myself, but I don’t think it is a big deal. We can always add `1-success_ratio` as failure_ratio, in case its better contrast with success_ration helps understand that error_ratio is something else.

Reply:
a great idea about failure ratio!

Reply:
would it be ok to add failure ratio for scrapy-zyte-api stats only?

Reply:
I am also OK with not adding it at all. As I said, I don‘t think the potential misinterpretation of error_ratio is a big deal, and we can probably also approach this with documentation _if_ people even complain.

Reply:
either approach is fine with me